uber/utils.py

```diff
@@ -262,6 +262,12 @@ def charge_cc(self, token):
             )
         except stripe.CardError as e:
             return 'Your card was declined with the following error from our processor: ' + str(e)
+        except stripe.error.InvalidRequestError as e:
+            log.error('Invalid source object provided the following error from our processor: ' + str(e), exc_info=True)
+            send_email(c.ADMIN_EMAIL, [c.ADMIN_EMAIL, 'dom@magfest.org'], 'MAGFest Stripe invalid request error',
+                       'Got an error while calling charge_cc(self, token={!r}):\n{}'
+                       .format(token, traceback.format_exc()))
+            return traceback.format_exc()
```

Comment:
This looks good overall, but we probably don't need to return the stacktrace to the client, since we're already logging and emailing it.